descriptors/ModuleDescriptor-template.json

```diff
@@ -170,6 +170,60 @@
         }
       ]
     },
+    {
+      "id": "finance.ledgers",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/ledgers",
+          "permissionsRequired": ["finance.ledgers.collection.get"],
+          "modulePermissions": [
+            "finance-storage.ledgers.collection.get"
+          ]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/finance/ledgers",
+          "permissionsRequired": ["finance.ledgers.item.post"],
+          "modulePermissions": [
+            "finance-storage.ledgers.item.post"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/ledgers/{id}",
+          "permissionsRequired": ["finance.ledgers.item.get"],
+          "modulePermissions": [
+            "finance-storage.ledgers.item.get"
+          ]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/finance/ledgers/{id}",
+          "permissionsRequired": ["finance.ledgers.item.put"],
+          "modulePermissions": [
+            "finance-storage.ledgers.item.put"
+          ]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/finance/ledgers/{id}",
+          "permissionsRequired": ["finance.ledgers.item.delete"],
+          "modulePermissions": [
+            "finance-storage.ledgers.item.delete"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/ledgers/{id}/groups",
+          "permissionsRequired": ["finance.group-fund-fiscal-years.collection.get"],
```

Comment:
I believe we need separate permission for this API, to be able give permission use  `GET /finance/group-fund-fiscal-year`, but restrict `GET /finance/ledgers/{id}/groups` usage or vice versa.

Reply:
removed this endpoint at all. Its implementation is out of scope of the story.